CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+### 3.4.1
+* Improve the RSpec Queue Mode runner log output (add seed)
```

Comment:
Please add a new line above.

Reply:
Please add below a link to the PR https://github.com/KnapsackPro/knapsack_pro-ruby/pull/178

This way, users can quickly access more relevant details about the release. Typically we have some context in the PR description with more details about what changed.